scriptmodules/supplementary/sdl2.sh

```diff
@@ -79,7 +79,7 @@ function build_sdl2() {
         conf_flags+=("--disable-video-x11")
     fi
     ! isPlatform "x11" && conf_flags+=("--disable-video-vulkan")
-    isPlatform "mali" && conf_flags+=("--enable-video-mali" "--disable-video-opengl")
+    isPlatform "mali" || isPlatform "vero4k" && conf_flags+=("--enable-video-mali" "--disable-video-opengl")
     isPlatform "rpi" && conf_flags+=("--enable-video-rpi")
```

Comment:
Why doesn't the previous line handle this? Is the Mali flag not set for vero4k? If not that needs to be changed there not here.